django-stubs/core/management/commands/inspectdb.pyi

```diff
@@ -10,7 +11,9 @@ class Command(BaseCommand):
     def normalize_col_name(
         self, col_name: str, used_column_names: List[str], is_relation: bool
     ) -> Tuple[str, Dict[str, str], List[str]]: ...
-    def get_field_type(self, connection: Any, table_name: Any, row: Any) -> Tuple[str, Dict[str, str], List[str]]: ...
+    def get_field_type(
+        self, connection: BaseDatabaseWrapper, table_name: Any, row: Any
```

Comment:
Is `table_name` really `Any`?

Reply:
Looks like `str` to me

Reply:
Yeah, I wanted to keep the scope down to `connection` arguments rather than chase every rabbit hole. This PR already took longer than I was hoping. But I'll add the hint for `table_name`.